_dashboards/management/multi-data-sources.md

```diff
@@ -73,18 +72,19 @@ To delete the data source connection, select the trash can icon ({::nomarkdown}<
 
 Use index patterns to access the OpenSearch data that you want to explore. Learn how to load your own data and create an index pattern in the following steps. This tutorial uses the preconfigured index pattern `opensearch_dashboards_sample_data_ecommerce Default`.
 
-1. In the Dashboards console, select **Index Patterns** and then **Create index pattern**.
-2. Select **Use external data source connection**.
-3. Start typing in the **Search data sources** field to search for the data source you created earlier and then select the data source and **Next step**.
-1. In the Dashboards console, choose **Index Patterns** and then **Create index pattern**.
-2. Choose **Use external data source connection**.
-3. Start typing in the **Search data sources** field to search for the data source you created earlier, select the data source, and then select **Next step**.
-4. Add an **Index pattern name** to define the index pattern and then choose **Next step**.
-5. Choose an option for the **Time field** and then select **Create index pattern**.
+1. Go to OpenSearch Dashboards, and select **Management** > **Dashboards Management** > **Index patterns**. 
+2. Select **Create index pattern**.
+3. Select **Use external data source connection**.
+4. Start typing in the **Search data sources** field to search for the data source you created earlier and then select the data source and **Next step**.
+5. In the Dashboards console, choose **Index Patterns** and then **Create index pattern**.
```

Comment:
Out of scope for this PR, but the usage of "Dashboards console" in this doc is confusing. Are we talking about the side navigation menu?